index.html

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body>
+    <div>
+        <p>Primeira Div</p>
+    </div>
+    <div>
+        <p>Segunda div</p>
+    </div>
+    <div>
+        <h1>Terceira div</h1>
```

Comment:
Acho que pode padronizar tudo em <p>.
```suggestion
        <p>Terceira div</p>
```